src/commands/fight.js

```diff
@@ -124,12 +128,19 @@ function fight(message, player1, player2, turn) {
 
     }).catch(() => {
         message.channel.send(`**${currentPlayer.user.username}** didn't respond, so we can assume that he has lost his turn.`);
+        currentPlayer.miss++;
         fight(message, player1, player2, !turn);
+        if (currentPlayer.miss === 2) {
+            message.channel.send(':x: Looks like No one is Responding, Terminating the game')
```

Comment:
Please change this to `':x: Looks like no one is responding, terminating the game.'` (fixed capitalization)